spring-kafka-2/src/test/java/com/baeldung/spring/kafka/managingkafkaconsumergroups/ManagingConsumerGroupsIntegrationTest.java

```diff
@@ -0,0 +1,57 @@
+package com.baeldung.spring.kafka.managingkafkaconsumergroups;
+
+import org.apache.commons.lang3.RandomUtils;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.kafka.config.KafkaListenerEndpointRegistry;
+import org.springframework.kafka.core.KafkaTemplate;
+import org.springframework.kafka.listener.MessageListenerContainer;
+import org.springframework.kafka.test.context.EmbeddedKafka;
+
+import java.util.Objects;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@SpringBootTest(classes = ManagingConsumerGroupsApplicationKafkaApp.class)
+@EmbeddedKafka(partitions = 2, brokerProperties = {"listeners=PLAINTEXT://localhost:9092", "port=9092"})
+public class ManagingConsumerGroupsIntegrationTest {
+
+    private static final String consumer1Identifier = "org.springframework.kafka.KafkaListenerEndpointContainer#1";
+    private static final int totalProducedMessages = 50000;
+    private static final int messageWhereConsumer1LeavesGroup = 10000;
```

Comment:
Just a style issue: The naming convention for constants is uppercase with words separated by underscores.